### DISCUSSION TITLE
What's next for Payload? Q2 2023 Edition

### DISCUSSION THREAD

**Discussion by jmikrut** [payloadcms/payload#2473] [General]
Hey everyone, I just took a second to write up a blog post about what we've got planned for this quarter. I'd love to hear from you all about what you think! Are we identifying our priorities right? Are we missing anything? Anything you'd like to see added? 

https://payloadcms.com/blog/q2-2023-whats-next

Thank you!

**Comment by kariyngva** [comment]
Really happy to see migrations on the list for Q2 🥳

**Reply by JG-devo** [reply]
Would love to see a few more built in security features, like implementing some form of official 2FA that can be enabled easily.
